app/src/main/java/com/keylesspalace/tusky/BaseActivity.java

```diff
@@ -125,7 +127,8 @@ private boolean activityTransitionWasRequested() {
 
     @Override
     protected void attachBaseContext(Context newBase) {
-        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(newBase);
+        // injected preferences not yet available at this point of the lifecycle
+        SharedPreferences preferences = ((TuskyApplication)newBase.getApplicationContext()).getPreferences();
```

Comment:
Consider using a Dagger [EntryPoint](https://dagger.dev/hilt/entry-points.html) instead.

Reply:
oh that is nice, will do